### ISSUE TITLE
Incorrect version displayed with `flutter` package compatibility

### ISSUE THREAD

**Issue by AlexV525** [dart-lang/pana#1128]
https://pub.dev/packages/photo_manager/score

![image](https://user-images.githubusercontent.com/15884415/190962729-ce3b06d0-7b8e-4e5b-b6ba-0f30bce498fd.png)

**Comment by sigurdm** [comment]
Yeah it is a bit confusing, there is a package called flutter (distinct from the sdk), and it always has v. 0.0.0. But it is coming from the flutter sdk (it is not hosted on pub.dev).
Not sure if we should remove the package from the view or show something else - but I agree the version number is not really meaningful.

**Comment by sigurdm** [comment]
We should probably leave out all SDK packages in the table

**Comment by sigurdm** [comment]
This is still relevant

We should probably leave out all non-hosted packages in the table

**Comment by sigurdm** [comment]
Should be fixed by doing https://github.com/dart-lang/pub/issues/4540

**Comment by sigurdm** [comment]
I was wrong before, `pub outdated` doesn't show sdk packages.

But pana passes `--up-to-date` which makes it do so: https://github.com/dart-lang/pana/blob/fe8716f64c5b29c8343c110eb73a8006a8ee6cac/lib/src/package_context.dart#L319

Most likely we want pana to filter the dependencies on the source type.
